src/lxc/lxccontainer.c

```diff
@@ -934,10 +906,12 @@ static bool do_lxcapi_start(struct lxc_container *c, int useinit, char * const a
 		return false;
 
 	if (!argv) {
+		char *cfgcmd = conf->init_cmd
```

Comment:
Someone's been doing golang :)

Reply:
so much so that i did not even recognize any of the 3 times i did that in 4 new lines of code. ;;;;